cluster/manifests/audittrail-adapter/credentials.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Cluster.ConfigItems.audittrail_url }}
+{{- if or (ne .Cluster.ConfigItems.audittrail_url "") (ne .Cluster.ConfigItems.audittrail_nakadi_url "") }}
```

Comment:
Would that also work? Based on the old version empty string seems to be `false`.
```
{{- if or .Cluster.ConfigItems.audittrail_url .Cluster.ConfigItems.audittrail_nakadi_url }}
```

Reply:
Yes, that should also work (not 100% sure if the parentheses are still required for the `or` though, but I don't think so without the strict comparison).

Reply:
Updated to use the simplified formatting.